src/graphql/Applicant/Types/Applicant.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLObjectType } from "graphql";
+import { ID, Int, nonNull, String, List } from "../../fieldTypes";
+import { GraphQLCapability } from "./Capability";
+import { GraphQLCareer } from "../../Career/Types/Career";
+
+const GraphQLApplicant = new GraphQLObjectType({
+  name: "Applicant",
+  fields: () => ({
+    uuid: {
+      type: ID
```

Comment:
(pregunta) con poner ID se asume que es not null ya? o habria que ponerlo?

Reply:
Emm.. no, me lo olvide. Hay que ponerlo